src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```diff
@@ -47,14 +46,8 @@ public WebHostBuilder()
                 ?? Environment.GetEnvironmentVariable("Hosting:Environment")
                 ?? Environment.GetEnvironmentVariable("ASPNET_ENV"));
 
-            if (Environment.GetEnvironmentVariable("Hosting:Environment") != null)
-            {
-                Console.WriteLine("The environment variable 'Hosting:Environment' is obsolete and has been replaced with 'ASPNETCORE_ENVIRONMENT'");
-            }
-            if (Environment.GetEnvironmentVariable("ASPNET_ENV") != null)
-            {
-                Console.WriteLine("The environment variable 'ASPNET_ENV' is obsolete and has been replaced with 'ASPNETCORE_ENVIRONMENT'");
-            }
+            // Add the default server.urls key
+            UseSetting(WebHostDefaults.ServerUrlsKey, Environment.GetEnvironmentVariable("ASPNETCORE_URLS"));
```

Comment:
I thought `urls` was going to become the primary key over here: https://github.com/aspnet/Hosting/blob/abe6dd5692c0d8bda573a25cdeb708a908dce517/src/Microsoft.AspNetCore.Hosting/Internal/WebHost.cs#L191
And then `server.urls` would become the obsolete fallback.